src/main/java/com/github/wovnio/wovnjava/WovnServletFilter.java

```diff
@@ -41,7 +41,11 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         if (((HttpServletResponse)response).containsHeader("X-Wovn-Handler")) {
             isRequestAlreadyProcessed = true;
         } else {
-            ((HttpServletResponse)response).setHeader("X-Wovn-Handler", "wovnjava_" + Settings.VERSION);
+            if (this.settings.showVersion) {
```

Comment:
I would make a method to generate the value:
```
    private String getWovnHandlerHeaderValue() {
        if (this.settings.showVersion) {
            return "wovnjava_" + Settings.VERSION;
        } else {
            return "wovnjava"
        }
    }
```

Also generate it inside `init` so we only do it once
`this.wovnHandlerHeaderValue = this.getWovnHandlerHeaderValue(settings)`